js/runner.go

```diff
@@ -172,6 +172,11 @@ func (r *Runner) newVU() (*VU, error) {
 		BPool:          bpool.NewBufferPool(100),
 	}
 	vu.Runtime.Set("console", common.Bind(vu.Runtime, vu.Console, vu.Context))
+	common.BindToGlobal(vu.Runtime, map[string]interface{}{
+		"open": func() {
+			common.Throw(vu.Runtime, errors.New("open function it is only available at init code"))
```

Comment:
I'd rephrase the error message as:
`"open" function is only available to the init code (aka global scope), see https://docs.k6.io/docs/test-life-cycle for more information`

I think including a link to the docs is helpful, but it of course adds some burden on us to make sure those links keep working if we change the URL